cli_reference/get_started_cli.adoc

```diff
@@ -10,13 +10,19 @@
 toc::[]
 
 == Basic Setup and Login
-The recommended way to setup the OpenShift CLI is using the `osc login` command. This command can be considered *the entry-point for every first-time OpenShift CLI user*. It features an *interactive flow* that will guide the user through establishing a session against an OpenShift server with the provided credentials, and will *automatically save* a config file that will be used by every subsequent command. 
+The recommended way to setup the OpenShift CLI is using the `osc login` command.
+This command can be considered *the entry-point for every first-time OpenShift
+CLI user*. It features an *interactive flow* that guides the user through
+establishing a session against an OpenShift server with the provided
+credentials. It then *automatically saves* a configuration file that is used by
+every subsequent command.
 
```

Comment:
Why is some of the text unnecessarily in *bold*? 

[Suggestion]
The `osc login` command is the best way to set up and configure the OpenShift CLI, and it serves as the entry point for most users. The interactive flow helps you establish a session to an OpenShift server with the provided credentials. The configuration is automatically saved and is then used for every subsequent command.